### DISCUSSION TITLE
ExplorerPatcher crashing multiple times on startup - CMD Prompt method not registering

### DISCUSSION THREAD

**Discussion by Firehawk894** [valinet/ExplorerPatcher#3924] [Q&A] [answered]
Hi all, sorry to be a bother.

I noticed EP crashing a bunch when I loaded up today, threw me the regular error message. I tried doing what the maker said by pasting this into Cmd Prompt/Powershell

Add-MpPreference -ExclusionPath "C:\Program Files\ExplorerPatcher"
Add-MpPreference -ExclusionPath "$env:APPDATA\ExplorerPatcher"
Add-MpPreference -ExclusionPath "C:\Windows\dxgi.dll"
Add-MpPreference -ExclusionPath "C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyewy"
Add-MpPreference -ExclusionPath "C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyewy"

But it gives me an error saying " 'Add-MpPreference' is not recognized as an internal or external command,
operable program or batch file."

Any idea how I can get around this so I don't have to use the eye torture that is Win11?

**Comment by pyrates999** [comment, marked as answer]
right click on powershell icon in start menu and select run as administrator.  Then you can run those commands.

**Reply by Firehawk894** [reply]
Huh, weird how it didn't work the first couple of times I tried it, maybe I had a space in there somewhere, thanks o/
